examples/stable/test/end_to_end/candid_rpc/candid_backtraces/benchmarks.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Get rid of this benchmark file, do not run the benchmarks and do not commit anything to do with benchmarks.